File: src/main/java/net/clgd/ccemux/rendering/awt/MouseTranslator.java
package net.clgd.ccemux.rendering.awt;

import static java.awt.event.MouseEvent.*;

public class MouseTranslator {
	public static int swingToCC(int button) {
		switch (button) {
			case BUTTON1:
				return 1;
			case BUTTON2:
				return 3;
			case BUTTON3:
				return 2;
			default:
				return -1;
		}
	}

	public static int ccToSwing(int button) {
		switch (button) {
			case 1:
				return BUTTON1;
			case 2:
				return BUTTON3;
			case 3:
				return BUTTON2;
			default:
				return -1;
		}
	}
}

File: src/main/kotlin/net/clgd/ccemux/rendering/awt/MouseTranslator.kt
package net.clgd.ccemux.rendering.awt

import java.awt.event.MouseEvent

object MouseTranslator {
	fun swingToCC(button: Int): Int {
		return when(button) {
			MouseEvent.BUTTON1 -> 1
			MouseEvent.BUTTON2 -> 3
			MouseEvent.BUTTON3 -> 2
			else -> -1
		}
	}

	fun ccToSwing(button: Int): Int {
		return when(button) {
			1 -> MouseEvent.BUTTON1
			2 -> MouseEvent.BUTTON3
			3 -> MouseEvent.BUTTON2
			else -> -1
		}
	}
}
